libsolidity/analysis/OverrideChecker.cpp

```diff
@@ -781,6 +781,14 @@ void OverrideChecker::checkOverrideList(OverrideProxy _item, OverrideProxyBySign
 			_item.astNodeNameCapitalized() + " has override specified but does not override anything."
 		);
 
+	if (_item.contract().isInterface() && !expectedContracts.empty())
+		overrideListError(
+			_item,
+			expectedContracts,
+			"An interface cannot override function in ",
```

Comment:
```suggestion
			"An interface cannot override function from ",
```
The printed one is the source of the overridden function, isn't it?

Reply:
Also, it then says `from contract...`. It'd be more precise if it says `from interface...`

Reply:
Having it say "contract" was the path of least resistance since that's what `overrideListError` already says. I'll try to find a clean way to change this.